### PR TITLE
Fix for custom sent death alerts

### PR DESCRIPTION
**Pull request by aaronma37**

### CODE REVIEW

File: Hardcore.lua [hunk 1]
```diff
@@ -2343,10 +2366,21 @@ function Hardcore:CHAT_MSG_ADDON(prefix, datastr, scope, sender)
 		end
 
 		if command == COMM_COMMANDS[10] then -- Received request for guild members
-			-- receiveDeathMsg(data, sender, command) would duplicate for sender
-			local commMessage = COMM_COMMANDS[11] .. COMM_COMMAND_DELIM .. data
-			CTL:SendAddonMessage("ALERT", COMM_NAME, commMessage, "GUILD")
-			return
+			local other_player_name = ""
+			local level = 0
+			local zone = ""
```
Comment (aaronma37) [inline Hardcore.lua:2371]: rm; unused

**Review by blenderer — APPROVED** [review]
LGTM

File: Hardcore.lua (hunk 1 above)
Comment (zdeyn) [reply Hardcore.lua:2371]: could we send the location as an id, and look up client side? we keep info, one less string to hack

Comment (aaronma37) [reply Hardcore.lua:2371]: That sounds good

**Pull request merged**